server/src/main/java/com/example/tyfserver/member/exception/AccountRegisteredException.java

```diff
@@ -0,0 +1,12 @@
+package com.example.tyfserver.member.exception;
+
+import com.example.tyfserver.common.exception.BaseException;
+
+public class AccountRegisteredException extends BaseException {
```

Comment:
```suggestion
public class AccountAlreadyRegisteredException extends BaseException {
```
`Already`를 붙이는게 이해하기 쉬울거같아!